### ISSUE TITLE
Geth does not start

### ISSUE THREAD

**Issue by wellgamer789** [bnb-chain/bsc#1849]
v1.2.11_linux

My command:
./geth --config ./config.toml --datadir /data/bsc_01/node --syncmode=full --disablevoteattestation --tries-verify-mode none --pruneancient=true --diffblock=10000 --txlookuplimit 0 --cache 49152 --ws --ws.addr 0.0.0.0 --ws.origins '*' --ws.api eth,web3 console

![image](https://github.com/bnb-chain/bsc/assets/29676107/8ccf0c5c-40c9-49b6-8611-32fc241f33a1)

`
t=2023-09-03T17:00:39+0200 lvl=info msg="Initialised chain configuration"    config="{ChainID: 56 Homestead: 0 DAO: <nil> DAOSupport: false EIP150: 0 EIP155: 0 EIP158: 0 Byzantium: 0 Constantinople: 0 Petersburg: 0 Istanbul: 0, Muir Glacier: 0, Ramanujan: 0, Niels: 0, MirrorSync: 5184000, Bruno: 13082000, Berlin: 31302048, YOLO v3: <nil>, CatalystBlock: <nil>, London: 31302048, ArrowGlacier: <nil>, MergeFork:<nil>, Euler: 18907621, Gibbs: 23846001, Nano: 21962149, Moran: 22107423, Planck: 27281024,Luban: 29020050, Plato: 30720096, Hertz: 31302048, Engine: parlia}"

t=2023-09-03T17:00:39+0200 lvl=info msg="Deep froze chain segment"           blocks=18 elapsed=1.385ms number=31,332,392 hash=0xf4bb1b6f5f9fc2dd17c82222d776b9105725b3e27a5ed058c5b7d2f2caf177be

t=2023-09-03T17:00:40+0200 lvl=info msg="Initialising Ethereum protocol"     network=56 dbversion=8

t=2023-09-03T17:00:40+0200 lvl=eror msg="Invalid block body RLP"             hash=0x90b473825dd5be95a92eb71c6bb9026ad11041be568626869d8b64b2e29db1bd err="rlp: value size exceeds available input length"

t=2023-09-03T17:00:40+0200 lvl=warn msg="Head block missing, resetting chain" hash=0x90b473825dd5be95a92eb71c6bb9026ad11041be568626869d8b64b2e29db1bd

t=2023-09-03T17:00:40+0200 lvl=warn msg="Rewinding blockchain"                target=0
`

https://github.com/bnb-chain/bsc/issues/1845#issue-1873560747

**Comment by CocoStarZ** [comment]
could you try without parameter  --diffblock=10000?

**Comment by wellgamer789** [comment]
> could you try without parameter --diffblock=10000?

Same problem

**Comment by CocoStarZ** [comment]
can't reproduce your problem. Did you sync from snapshot or genesis?

**Comment by wellgamer789** [comment]
> can't reproduce your problem. Did you sync from snapshot or genesis?

My nodes that were running on 1.2.9 were all with less than 20 peers right after the hardfork, when i updated to 1.2.11 and couldn't run anymore. Then i cleaned everything and set it to sync from the snapshot provided by bnb48, the same problem happened of running out of peer and when i tried to restart the same thing happened of me not being able to run it. I tried to sync the genesis in a other machine, but it's been days and until today it didn't arrive in the current block.

Making it clear that all this happened right after the last hardfork

**Comment by Classical1956** [comment]
> > can't reproduce your problem. Did you sync from snapshot or genesis?
> 
> My nodes that were running on 1.2.9 were all with less than 20 peers right after the hardfork, when i updated to 1.2.11 and couldn't run anymore. Then i cleaned everything and set it to sync from the snapshot provided by bnb48, the same problem happened of running out of peer and when i tried to restart the same thing happened of me not being able to run it. I tried to sync the genesis in a other machine, but it's been days and until today it didn't arrive in the current block.
> 
> Making it clear that all this happened right after the last hardfork

I think your node has been specially modified (magic-geth),
Had the same problem but I solved it.

**Comment by brilliant-lx** [comment]
if it is problem of finding useful peers, you may try the latest release v1.2.12, it provides bootnodes.

**Comment by wellgamer789** [comment]
> if it is a problem of finding useful peers, you may try the latest release v1.2.12, it provides bootnodes.

I updated and the first thing is that i can't start the node due to the same problem as above.

**Comment by wellgamer789** [comment]
I was running 1.2.9 for a long time, right after the last hardfork my number of peers dropped to 10 and they are not constant, they keep disconnecting and new ones arriving.

I stopped the node and updated it to 1.2.11, at that moment the node no longer started and the error that I presented above appeared.

I cleaned the db and started from a snapshot of the BNB48 club, when it was synchronizing it was full of peers, but when it reached the current block the number of peers quickly dropped until it was left with few peers. I stopped and started the node again, i received the same error as before.

I tried with 1.1.12 and it was the same thing.
